source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -133,6 +133,11 @@ def _direct_path(
             mode, pt_object_origin, pt_object_destination, fallback_extremity, direct_path_request
         )
 
+        # Ask the PO, what do we want really?
+        if direct_path_type != StreetNetworkPathType.DIRECT and mode == 'car':
+            req.direct_path.streetnetwork_params.origin_mode = 'car_no_park'
```

Comment:
*car_no_park* mode already exist in kraken ?

Reply:
yes, it is an `internal` mode used for ridesharing and taxi for kraken, but it's not yet handled by asgard